codespell_lib/_codespell.py

```diff
@@ -54,7 +54,7 @@
     # realistic for obscure words
     ('clear', 'for unambiguous errors', '',
         False, None, supported_languages_en, None),
-    ('rare', 'for rare but valid words', '_rare',
+    ('rare', 'to categorize rare but valid words as errors', '_rare',
```

Comment:
How about one of these?:
```suggestion
    ('rare', 'for rare (but valid) words which are likely to be errors', '_rare',
```
```suggestion
    ('rare', 'for rare (but valid) words which are likely to be errors where someone meant the correction', '_rare',
```

I would say we're not really categorising. Also can you try running the --help, I think it uses this output and not having for will make for some odd phrasing.